### PR TITLE
Add managing, monitoring, and tuning sections

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
Signed-off-by: Naarcha-AWS <naarcha@amazon.com>

### Description

Fixes merge conflicts from #2353 

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by hdhalter — APPROVED** [review]
Looks good, Nate! I did some random link checking and didn't find any issues.

File: _tuning-your-cluster/availability-and-recovery/snapshots/sm-api.md [hunk 1]
```diff
@@ -1,14 +1,17 @@
...
 has_children: false
+grand_parent: Availability and Recovery
+redirect_from: 
+  - /opensearch/snapshots/sm-api/
 ---
 
 # Snapshot Management API
 
-Use the [Snapshot Management (SM)]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots) API to automate [taking snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots). 
+Use the [snapshot management (SM)]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots) API to automate [taking snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots).
```
Comment (cwillum) [inline _tuning-your-cluster/availability-and-recovery/snapshots/sm-api.md:14]: Why do two separate links in the same sentence take you to the same place?

File: about.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ OpenSearch is a distributed search and analytics engine based on [Apache Lucene]
...
 
-An equally popular, but less obvious use case is log analytics, in which you take the logs from an application, feed them into OpenSearch, and use the rich search and visualization functionality to identify issues. For example, a malfunctioning web server might throw a 500 error 0.5% of the time, which can be hard to notice unless you have a real-time graph of all HTTP status codes that the server has thrown in the past four hours. You can use [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/) to build these sorts of visualizations from data in OpenSearch.
+An equally popular, but less obvious use case is log analytics, in which you take the logs from an application, feed them into OpenSearch, and use the rich search and visualization functionality to identify issues. For example, a malfunctioning web server might throw a 500 error 0.5% of the time, which can be hard to notice unless you have a real-time graph of all HTTP status codes that the server has thrown in the past four hours. You can use [OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/dashboards/index/) to build these sorts of visualizations from data in OpenSearch.
```
Comment (cwillum) [inline about.md:20]: I can't find this file.
I don't see the /dashboards/index/ link.

**Review by cwillum — APPROVED** [review]
With some comments.

File: _tuning-your-cluster/availability-and-recovery/snapshots/sm-api.md (hunk 1 above)
Comment (Naarcha-AWS) [reply _tuning-your-cluster/availability-and-recovery/snapshots/sm-api.md:14]: I'll remove the first.

File: about.md (hunk 2 above)
Comment (Naarcha-AWS) [reply about.md:20]: This original index URL redirects to here: https://opensearch.org/docs/latest/dashboards/quickstart-dashboards/

**Pull request closed without merging**